src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java

```diff
@@ -18,20 +18,21 @@
 import net.minecraft.potion.PotionEffect;
 import net.minecraft.util.MathHelper;
 import net.minecraft.util.ResourceLocation;
+import org.jetbrains.annotations.Contract;
 import org.lwjgl.opengl.GL11;
 
 import java.util.concurrent.ConcurrentLinkedQueue;
 import java.util.concurrent.ThreadLocalRandom;
 
 public class VictoryRoyale extends AbstractMod {
-
     private static VictoryRoyale INSTANCE;
 
     private ResourceLocation texture = new ResourceLocation("textures/material/victoryroyale/victory_royale.png");
     private ConcurrentLinkedQueue<WhiteLine> points = new ConcurrentLinkedQueue<>();
 
     private long start = 0;
 
+    @Contract(pure = true)
```

Comment:
Why is this necessary?

Reply:
Sorry, IntelliJ has a mind of its own.  Removed.